src/tool-myank.cpp

```diff
@@ -246,6 +249,14 @@ void Tool_myank::processFile(HumdrumFile& infile) {
 	getMeasureStartStop(MeasureInList, infile);
 
 	string measurestring = getString("measures");
+
+	if(getBoolean("lines")) {
```

Comment:
There should be a space between `if` and the following `(` (because `if` is a keyword and not a function).